tools/Ergo_Proxy_Windows.ps1

```diff
@@ -0,0 +1,22 @@
+﻿#Change proxy address script
+set-itemproperty -path "hkcu:Software\Microsoft\Windows\CurrentVersion\Internet Settings" -name AutoConfigURL -value "http://127.0.0.1:2000/proxy.pac" -type string
+
```

Comment:
This is already done in setup: https://github.com/cristianoliveira/ergo/blob/master/commands/setup.go#L71